conf/app.routes

```diff
@@ -34,17 +34,25 @@ POST       /access-code                             controllers.AccessCodeContro
 GET        /change-access-code                      controllers.AccessCodeController.onPageLoad(mode: Mode = CheckMode)
 POST       /change-access-code                      controllers.AccessCodeController.onSubmit(mode: Mode = CheckMode)
 
-GET        /check-your-answers                      controllers.CheckYourAnswersController.onPageLoad()
-POST       /check-your-answers                      controllers.CheckYourAnswersController.onSubmit()
+GET        /check-your-answers                      controllers.CheckYourAnswersControllerV1.onPageLoad()
+POST       /check-your-answers                      controllers.CheckYourAnswersControllerV1.onSubmit()
 
-GET        /try-again                               controllers.TryAgainController.onPageLoad()
-POST       /try-again                               controllers.TryAgainController.onSubmit()
+GET        /check-answers                           controllers.CheckYourAnswersControllerV2.onPageLoad()
+POST       /check-answers                           controllers.CheckYourAnswersControllerV2.onSubmit()
+
+GET        /try-again                               controllers.TryAgainControllerV1.onPageLoad()
+POST       /try-again                               controllers.TryAgainControllerV1.onSubmit()
+
+
+GET        /try-again                               controllers.TryAgainControllerV2.onPageLoad()
+POST       /try-again                               controllers.TryAgainControllerV2.onSubmit()
```

Comment:
Will identical URLs not mean this will always redirect to whichever one is defined first? In this case the V1 version

Reply:
I did initially think that @jackheslop96 however, it seems to test OK switching between V1 and V2 modules. Maybe a little more testing around this. If this does turn out to be the case we will need a different url for V2 or a different approach.